Docs/user_guide/modules/index.md

```diff
@@ -170,3 +170,7 @@ brainslabelstats.md
 performancetests.md
 selftests.md
 ```
+
+## Retired Modules
+
+See the list of [retired modules](retiredmodules.md).
```

Comment:
@lassoan In this index file you are suggesting it be something more like the following?

```
## Retired Modules
```{toctree}
:maxdepth: 1
retiredmodules.md
```

Reply:
Feel free to commit directly to this PR any suggested changes here. I'm signing off working for today.

Reply:
> In this index file you are suggesting it be something more like the following?

Yes, it should be added into a toctree.

Reply:
It's unclear to me what to do here as we aren't listing retired module pages like how the other modules are listed in categories. We aren't keeping retired module documentation, but linking to another page (retiredmodules.md) from this index page that details the removed modules in a table of information.

Reply:
My only problem is that when you open [this page](https://slicer--6039.org.readthedocs.build/en/6039/user_guide/modules/retiredmodules.html) then nothing is selected in the document tree on the left.

![image](https://user-images.githubusercontent.com/307929/143912920-1986d91e-080a-40b7-930c-a2dccb597806.png)

Reply:
Yeah I don't know what to do about that. I could as a workaround not have an individual retiredmodules.md page and instead put all the content of that page at the bottom of the index.md page.

Reply:
OK, I'll try to fix it then. Don't make forced pushes until I commit a change.
Was this the last open discussion?

Reply:
Ok sounds good. Yes this is the last open discussion.